src/Dev/Deprecation.php

```diff
@@ -208,7 +237,11 @@ public static function isEnabled(): bool
         if (!Director::isDev()) {
             return false;
         }
-        return static::$currentlyEnabled || Environment::getEnv('SS_DEPRECATION_ENABLED');
+        $envVar = Environment::getEnv('SS_DEPRECATION_ENABLED');
+        if ($envVar !== null) {
+            return (bool) $envVar;
+        }
+        return static::$currentlyEnabled;
```

Comment:
I realised that the "This will be overriden by the SS_DEPRECATION_ENABLED environment variable if present" promise in the PHPDoc of the private static (and now also in the enable/disable methods) wasn't being kept if, say, someone set
```sh
SS_DEPRECATION_ENABLED=0
```
to _explicitly disable_ deprecation warnings.